public/javascripts/songForm.js

```diff
@@ -3,7 +3,8 @@
 //url is the url for post method
 //readonly is a flag to set whether the input readonly or not
 //submitButton is a flag to determine whether to show submit button at the bottom of the form or not
-var songForm = function(song, url, readonly, submitButton) {
+//formID is the id for differentiation all element in orisong form and translation form
+function songForm(song, url, readonly, submitButton, formID) {
```

Comment:
once you have more than 3 arguments, it's preferable to pass in a single object as the argument:
<code>
{
  song: song,
  url: url, 
  readonly: false,
  submitButton: false,
  formID: 'yada'
}
</code>